versioned.js

```diff
@@ -46,7 +46,8 @@ const generateVersionedDocs = async (versions) => {
     const configFile = `./version/${version}.yml`
     const newSrc = `./version/src_${version}`
     versionedMkdocs.extra.versions = extraVersions
-    versionedMkdocs.docs_dir = newSrc
+    versionedMkdocs.docs_dir = `./src_${version}`
+    versionedMkdocs.theme.custom_dir = '../theme'
     fs.writeFileSync(`./version/${version}.yml`, YAML.stringify(versionedMkdocs, 7), 'utf8')
     fse.copySync(`./src`, newSrc)
     await git.checkout(returnToBranch)
```

Comment:
I'd consider clarifying the intent behind the steps of this code. It is absolutely not clear why it first switches to `master` (returnToBranch) and then builds the docs, and it works (it seems it works because it makes a copy of the sources). The fact there was a bug just confirmed it begs the clarifications of the intent.

Reply:
I have no idea tbh; I didn't write the script.

Reply:
Well, I am not saying "rewrite the whole thing" but "since we've already understood the intent, why don't we clarify it for future mainteiners — possibly us — through some implementation comments/clearer structure/etc so that they don't have to invest some time to uncover it each time it needs to be modified".